src/API/index.js

```diff
@@ -12,3 +12,7 @@ export const fetchInstanceTypesList = async (sourceId) => {
   );
   return data;
 };
+
+export const createAWSReservation = async (params) => {
+  return axios.post(provisioningUrl('reservations/aws'), params);
```

Comment:
are we doing any failure handling here, or is it todo?

Reply:
explained offline, @amirfefer explained we are doing it in sideffect, see below :)